recipes/greedy/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# A dummy greedy repo is cloned because CMakeLists.txt requires
+# .git dir
+
+# Clone the repository into a temporary directory
+git clone https://github.com/pyushkevich/greedy /tmp/greedy
+
+# Change to the directory where the source is located
+cd /tmp/greedy
```

Comment:
You should patch the sources rather than doing this.

Also the current implementation of this completely ignores the version number and just takes the latest version of the sources.

Reply:
Apologies for this oversight. I've removed these lines of codes as they were mainly experimental.

Reply:
@conda-forge/help-c-cpp ready for review please!